classes/migration/GenresMigration.inc.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * @file classes/migration/GenresMigration.inc.php
+ *
+ * Copyright (c) 2014-2020 Simon Fraser University
+ * Copyright (c) 2000-2020 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class GenresMigration
+ * @brief Describe database table structures.
+ */
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Builder;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Capsule\Manager as Capsule;
+
+class GenresMigration extends Migration {
+        /**
+         * Run the migrations.
+         * @return void
+         */
+        public function up() {
+		// A context's submission file genres.
+		Capsule::schema()->create('genres', function (Blueprint $table) {
+			$table->bigInteger('genre_id')->autoIncrement();
+			$table->bigInteger('context_id');
+			//  NOTNULL 
+			//  For upgrades; see bug #8585 
+			$table->bigInteger('seq')->nullable();
+			$table->tinyInteger('enabled')->default('1');
+			$table->bigInteger('category')->default('1');
+			$table->tinyInteger('dependent')->default('0');
```

Comment:
Is the default here supposed to be a string rather than an int?

Reply:
Ah, that came from `var_dump` in the conversion tool. I don't think it's important but I'll tweak it.